enhancements/installer/image-based-installer.md

```diff
@@ -0,0 +1,255 @@
+using automated tooling of their choice, the ISO format offers the widest range
+of compatibility. Building a single ISO to boot multiple hosts makes it
+considerably easier for the user to manage. The additional site configuration
+ISO is necessary for configuring each cluster securely and independently.
+
+The user, before running the image-based installer, must generate a [seed image](https://github.com/openshift-kni/lifecycle-agent/blob/main/docs/seed-image-generation.md)
+via the [Lifecycle Agent SeedGenerator Custom Resouce (CR)](https://github.com/openshift-kni/lifecycle-agent/blob/main/docs/seed-image-generation.md).
+The prerequisites to generating a seed image are the following:
+
+- An already provisioned single-node OpenShift cluster (seed SNO).
+   - The CPU topology of that host must align with the target host(s), i.e. they
+     should have the same number of cores.
+- The [Lifecycle Agent](https://github.com/openshift-kni/lifecycle-agent/tree/main)
+  operator must be installed on the seed SNO.
+
+### Risks and Mitigations
+
+N/A
+
+### Drawbacks
+
+N/A
+
+## Open Questions [optional]
+
+- Should the command-line tool be a subcommand of the OpenShift installer, or a
+  standalone binary?
+
+  Having the functionality provided by the command-line tool in the OpenShift
+  installer would be a natural addition to the latter, as the former refers to
+  the provisioning of single-node OpenShift clusters and generates the
+  required installation artifacts in the same way as the [Agent-based Installer](/enhancements/agent-installer/agent-based-installer.md).
+
+## Test Plan
+
+The image-based installer will be covered by end-to-end testing using virtual
+machines (in a baremetal configuration), automated by some variation on the
+metal platform [dev-scripts](https://github.com/openshift-metal3/dev-scripts/#readme).
+This is similar to the testing of the agent-based installer, the baremetal IPI
+and assisted installation flows.
+
+## Graduation Criteria
+
+**Note:** *Section not required until targeted at a release.*
+
+TBD
+
+### Dev Preview -> Tech Preview
+
+- Ability to utilize the enhancement end to end
+- End user documentation, relative API stability
+- Sufficient test coverage
+- Gather feedback from users rather than just developers
+- Enumerate service level indicators (SLIs), expose SLIs as metrics
+- Write symptoms-based alerts for the component(s)
+
+### Tech Preview -> GA
+
+- More testing (upgrade, downgrade, scale)
+- Sufficient time for feedback
+- Available by default
+- Backhaul SLI telemetry
+- Document SLOs for the component
+- Conduct load testing
+- User facing documentation created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+**For non-optional features moving to GA, the graduation criteria must include
+end to end tests.**
+
+### Removing a deprecated feature
+
+N/A
+
+## Upgrade / Downgrade Strategy
+
+N/A
+
+## Version Skew Strategy
+
+N/A
```

Comment:
In the implementation I have seen there is nothing to restrict the seed image, the RHCOS base ISO, the installation ignition, and the configuration ISO coming from 4 different versions of OpenShift.

So a version skew strategy is absolutely applicable.

Reply:
Updated, PTAL.